utils/permissions/src/main.rs

```diff
@@ -109,21 +110,23 @@ fn parse_permissions(doc: EnumDoc) -> Vec<(String, String)> {
             children
                 .iter()
                 .enumerate()
-                .filter(|(_, child)| h1_selector.matches(child))
-                // Check if the heading is `# Permissions`
+                .filter(|(_, child)| h1_selector.matches(child) || h2_selector.matches(child))
+                // Check if the heading is `Permissions`
                 .filter(|(_, child)| {
                     child
                         .value()
                         .id()
                         .map(|s| s.starts_with("permissions"))
                         .unwrap_or(false)
                 })
-                // Look for a ul in the contents before the next h1
+                // Look for a ul in the contents before the next h1/h2
                 .for_each(|(i, _)| {
                     children[i + 1..]
                         .iter()
                         // Stop once we hit another h1
-                        .take_while(|element| !h1_selector.matches(element))
+                        .take_while(|element| {
+                            !(h1_selector.matches(element) || h2_selector.matches(element))
```

Comment:
maybe extract the inner parens to a closure and use in the filter above as well?

Reply:
Tried that earlier but compiler started complaining about lifetimes and I went: not worth it :)

Reply:
Seems like it worked out in the end, neat :)

Reply:
Yeah but I had to deference the element which I believe will cause a shadow copy to be created so not ideal but whatever lol